mongo/integration/change_stream_test.go

```diff
@@ -586,6 +586,23 @@ func TestChangeStream_ReplicaSet(t *testing.T) {
 			assert.Equal(mt, 2, numClearedEvents, "expected two PoolCleared events, got %d", numClearedEvents)
 		})
 	})
+	mt.RunOpts("call to cursor.Next after cursor closed", mtest.NewOptions().MinServerVersion("4.0"), func(mt *mtest.T) {
+		cs, err := mt.Coll.Watch(mtest.Background, mongo.Pipeline{})
+		assert.Nil(mt, err, "Watch error: %v", err)
+		defer closeStream(cs)
+		// Generate insert events
+		generateEvents(mt, 5)
+		// Call Coll.Drop to generate drop and invalidate event
+		assert.Nil(mt, mt.Coll.Drop(mtest.Background), "expected Nil but returned %v", err)
```

Comment:
If the `Drop` fails, the error message in the will be the one from `Watch`. This should instead be organized as two separate lines:

```
err = mt.Coll.Drop(...)
assert.Nil(mt, err, "Drop error: %v", err)
```

Reply:
done